.github/workflows/deploy-to-control-plane.yml

```
@@ -194,102 +231,41 @@
-            
-            const failureMessage = [
-              ' Deployment failed for PR #' + prNumber + ', commit ' + '${{ env.COMMIT_HASH }}',
+              '🌐 Review app is ready at: ${{ env.REVIEW_APP_URL }}',
               '',
-              ' [View Deployment Logs with Errors](' + workflowUrl + ')',
+              '📝 [View App Logs](' + process.env.WORKFLOW_URL + ')',
               '',
-              getConsoleLink(prNumber)
+              getConsoleLink(process.env.PR_NUMBER)
             ].join('\n');
             
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)